tests_and_analysis/test/run_tests.py

```diff
@@ -1,9 +1,27 @@
+import argparse
 import sys
 import os
+import time
+from typing import Tuple, List, Union
+
 import pytest
 import coverage
-import argparse
-from typing import Tuple, List, Union
+
+
+def main():
+    test_dir, reports_dir = _get_test_and_reports_dir()
+
+    do_report_coverage, do_report_tests, tests, markers_to_run = \
+        _get_parsed_args(test_dir)
+
+    pytest_options: List[str] = _build_pytest_options(
```

Comment:
Minor inconsistency in format, purely cosmetic.

Reply:
Fixed.